pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -127,12 +112,39 @@ func (r *ReconcileAPIScheme) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, nil
 	}
 
-	ownerTags, err := utils.AWSOwnerTag(r.client)
+	// Does the Service exist already?
+	found := &corev1.Service{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: instance.Spec.ManagementAPIServerIngress.DNSName, Namespace: "openshift-kube-apiserver"}, found)
 	if err != nil {
-		reqLogger.Error(err, "Couldn't get the cluster owner tags")
-		SetAPISchemeStatus(instance, "Couldn't reconcile", "Couldn't get the cluster owner tags", cloudingressv1alpha1.ConditionError)
-		r.client.Status().Update(context.TODO(), instance)
-		return reconcile.Result{}, err
+		if errors.IsNotFound(err) {
+			// need to create it
+			dep := r.newServiceFor(instance)
+			err = r.client.Create(context.TODO(), dep)
```

Comment:
Does the new object need an owner reference?

Reply:
@mbarnes Strictly speaking it would be nice to have, but [cross-namespace ownership doesn't work](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents):

> *Note*: Cross-namespace owner references are disallowed by design. This means: 1) Namespace-scoped dependents can only specify owners in the same namespace, and owners that are cluster-scoped. 2) Cluster-scoped dependents can only specify cluster-scoped owners, but not namespace-scoped owners.